### DISCUSSION TITLE
classification predict output interpretation

### DISCUSSION THREAD

**Discussion by krisvive** [pycaret/pycaret#1112] [Q&A] [answered]
Hi All,

I am trying to solve a binary classification problem (True Vs False) using PyCaret (2.3.0). After evaluating multiple threshold values, I zeroed to one (0.4) and used it for predicting my final unseen data. I was expecting the '_Score_' above threshold should be '_Label_'ed as True and below Threshold as False. Referring to #215 , I understand that '_Score_' gives probability of positive outcome only (in my case, True) and to get probability of False, I need to do (1-_Score_)

Surprisingly, when I analyzed the _predict_model()_ output; it had scores only above threshold and that also with both labels (True and False). for e.g. records with 'score' > 0.4 were classified as True and False both. Attaching a plot (_plt.scatter(predictions["Label"].astype(str), predictions["Score"])_) for reference...
![image](https://user-images.githubusercontent.com/6905855/111434266-5b494400-8725-11eb-9b65-e7131d592783.png)

Can anyone help me understand if I am missing something here or not using _predict_model()_ output correctly? Very much appreciate your support

**Comment by Yard1** [comment, marked as answer]
You can do `raw_score=True` in `predict_model()` to receive the same output as you would from sklearn - perhaps that will help?

**Reply by krisvive** [reply]
Thanks @Yard1 , this solved my problem. I am now getting 2 scores, 'Score_True' and 'Score_False' with a cleaned Label field. Just a suggestion, shouldn't this be the default output of predict_model() instead of 'Label' having both classes, it might confuse the end-user who is new to PyCaret/ML

I was not aware of this parameter in predict_model(), also couldn't find it in the documentation. Is it there only with specific latest versions?

**Reply by Yard1** [reply]
It should be present in the readthedocs documentation
